libdirections-offboard/build.gradle

```diff
@@ -17,10 +17,15 @@ android {
 dependencies {
     implementation project(':libnavigation-util')
 
+    // Support
+    implementation dependenciesList.supportAppcompatV7
```

Comment:
Do we need this dependency here now?

Reply:
yep

Reply:
Let's use `dependenciesList.supportAnnotation` then, which is a much smaller dependency.

Reply:
good catch